lib/core/include/query_processor.hpp

```diff
@@ -23,12 +23,14 @@ namespace irods
         using errors     = std::vector<error>;
         using result_row = typename query<ConnectionType>::value_type;
         using job        = std::function<void (const result_row&)>;
+        using query_type = typename query<ConnectionType>::query_type;
```

Comment:
We should probably add `_type` to the rest of the type aliases since they are in the same class.

Reply:
I just realized that query_type is already in line with the other aliases. Its fine as it is.